src/reporting/market.se

```diff
@@ -430,7 +430,7 @@ def getShareToken(outcome: int256):
 
 def getFinalPayoutDistributionHash():
     assertNoValue()
-    return(self.finalPayoutDistributionHash)
+    return(self.finalPayoutDistributionHash : bytes32)
```

Comment:
Style: I think elsewhere we are doing `name: type` (no space before `:`).  I don't think we have discussed this, but it is inconsistent here even within this change.  Recommend either changing to `name: type` or starting a discussion around which style we want to go with as a team.